conda-recipes/kfac-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "kfac" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tensorflow/kfac 
+  git_rev: master
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - six
+    - numpy
+    - tensorflow-probability
+    - tensorflow-gpu >=1.14
```

Comment:
Is KFAC GPU only?

Reply:
This line says so https://github.com/tensorflow/kfac/blob/master/setup.py#L39. However, in the README, even cpu version is mentioned. I'll replace `tensorflow-gpu` with `tensorflow`.